Source/ISeriesProvider/DB2iSeriesSqlBuilder.cs

```diff
@@ -319,7 +325,7 @@ protected override string GetProviderTypeName(IDbDataParameter parameter)
 		//Use mapping schema and internal db datatype mapping information to get the appropriate dbType
 		protected override void BuildDataTypeFromDataType(SqlDataType type, bool forCreateTable)
 		{
-			var dbType = MappingSchema.GetDbDataType(type.SystemType, type.Type.DataType, type.Type.Length, type.Type.Precision, type.Type.Scale, forCreateTable, db2iSeriesSqlProviderFlags.SupportsNCharTypes);
+			var dbType = MappingSchema.GetDbDataType(type.SystemType, type.Type.DataType, type.Type.Length, type.Type.Precision, type.Type.Scale, false, db2iSeriesSqlProviderFlags.SupportsNCharTypes);
```

Comment:
That parameter eventually became the `forceDefaultAttributes` parameter of this function: https://github.com/devinlyons/Linq2DB4iSeries/blob/e8b2985bd786e2f999c43236b93fd371c7bcb6c2/Source/ISeriesProvider/DB2iSeriesDbTypes.cs#L155.

It caused the `CREATE TABLE` statements to ignore the precision parameter. This caused problems with the `TIMESTAMP` type for tests that expected it to have a precision of 7 because it defaults to 6. Basically, I was getting SQL like this:

`CREATE TABLE SCHEMA.TABLE_NAME ( FIELD_NAME TIMESTAMP )`

Now I get SQL like this:

`CREATE TABLE SCHEMA.TABLE_NAME ( FIELD_NAME TIMESTAMP(7) )`

Reply:
Ok i see, I don't remember if I had any other issues with other types that forced this to false or if it was an oversight. I guess if the tests pass it's ok.